File: thisboot/src/main/java/com/brageast/blog/thisboot/ThisbootApplication.java
package com.brageast.blog.thisboot;

import lombok.extern.slf4j.Slf4j;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * 开始的地方
 *
 * @author chenmo
 */
@Slf4j
@SpringBootApplication
public class ThisbootApplication {

    public static void main(String[] args) {
        SpringApplication.run(ThisbootApplication.class, args);
        log.info("代码托管于 https://github.com/chenmoand/thisme");
    }

}

File: thisboot/src/main/kotlin/com/brageast/blog/thisboot/ThisbootApplication.kt
package com.brageast.blog.thisboot

import com.brageast.blog.thisboot.util.loggerOf
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


val log = loggerOf<ThisbootApplication>()

@SpringBootApplication/*(exclude = [ReactiveUserDetailsServiceAutoConfiguration::class])*/
class ThisbootApplication

// enable spring boot
fun main(args: Array<String>) {
    runApplication<ThisbootApplication>(*args)
    log.info("代码托管于 https://github.com/chenmoand/thisme")
}

